efel/pyfeatures/pyfeatures.py

```diff
@@ -51,6 +53,35 @@ def time():
     return _get_cpp_feature("time")
 
 
+def impedance():
+    from scipy.ndimage.filters import gaussian_filter1d
+    voltage_trace = voltage()
+    holding_voltage = _get_cpp_feature("voltage_base")
+    normalized_voltage = voltage_trace - holding_voltage
+    current_trace = current()
+    if current_trace is not None:
+        current_base = numpy.median(current_trace[0:10])
```

Comment:
maybe this should be called holding_current to be consistent with holding_voltage?

Reply:
Done in latest commit